modules/helm-adding-namespace-scoped-helm-chart-repositories.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc
+
+:_content-type: PROCEDURE
+[id="adding-namespace-scoped-helm-chart-repositories.adoc_{context}"]
+= Adding namespace-scoped custom Helm chart repositories
+
+[role="_abstract"]
+The cluster-scoped `HelmChartRepository` custom resource definition (CRD) for Helm repository provides the ability for admins to add Helm repositories as custom resources. The namespace-scoped `ProjectHelmChartRepository` CRD allows project members with the appropriate RBAC permissions to create Helm repository resources of their choice but scoped to their namespace. Such project members can see charts from both cluster-scoped and namespace-scoped Helm repository resources.
```

Comment:
admins/administrators - throughout the PR
RBAC is spelled out in the note below and not here. Suggest reversing it. The first instance should be spelled out.